pages/api/events.js

```diff
@@ -12,14 +12,29 @@ export default async function handler(req, res) {
   return res.status(200).json(events);
 }
 
-export async function getEvents() {
+export async function getEvents(withLocation = false) {
```

Comment:
This not reachable by the API end point, a query parameter should be added to the `handler` function above

Reply:
Resolved here https://github.com/EddieHubCommunity/LinkFree/pull/8093/commits/04ff78e6197e5d1e83cf1143cf123fcf604b6d0a